### PR TITLE
Document contributor roles and add maintainers list

### PR DESCRIPTION
**Pull request by tatiana**
Add documentation about Astronomer Cosmos contributor roles and maintainers until now.

### CODE REVIEW

File: docs/contributors-roles.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Contributors
+------------
+
+A contributor is anyone who wants to contribute code, documentation, tests, ideas, or anything to the Astronomer Cosmos project.
+
+Cosmos contributors can be found in the Astronomer Cosmos Github `insights page <https://github.com/astronomer/astronomer-cosmos/graphs/contributors>`_ and in the `#airflow-dbt <https://join.slack.com/t/apache-airflow/shared_invite/zt-1zy8e8h85-es~fn19iMzUmkhPwnyRT6Q>`_ Slack channel.
+
+Contributors are responsible for:
+
+* Fixing bugs
```
Comment (tatiana) [inline docs/contributors-roles.rst:21]:
```suggestion
* Fixing bugs
* Refactoring code
```

File: docs/contributors-roles.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Committers are community members with write access to the `Astronomer Cosmos Github repository <https://github.com/astronomer/astronomer-cosmos>`_.
+They can modify the code and the documentation and accept others' contributions to the repo.
+
+Check :ref:`contributors` for the official list of Astronomer Cosmos committers.
+
+Committers have the same responsibilities as standard contributors and also perform the following actions:
+
+* Reviewing & merging pull-requests
+* Scanning and responding to GitHub issues, helping triaging them
+* Releasing new versions of Cosmos in `PyPI <https://pypi.org/project/astronomer-cosmos/>`_, following the `milestones <https://github.com/astronomer/astronomer-cosmos/milestones>`_.
```
Comment (tatiana) [inline docs/contributors-roles.rst:39]:
```suggestion
```

File: docs/contributors-roles.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Refactoring code
```
Comment (tatiana) [inline docs/contributors-roles.rst:40]:
```suggestion
```

File: docs/contributors-roles.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* Improving processes and tooling
+
+If you know you are not going to be able to contribute for a long time (for instance, due to a change of job or circumstances), you should inform other maintainers and we will mark you as "inactive".
+Inactive committers will no longer have write access to the repo.
```
Comment (tatiana) [inline docs/contributors-roles.rst:44]:
```suggestion
Emeritus committers will no longer have write access to the repo.
```

File: docs/contributors-roles.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Check :ref:`contributors` for the official list of Astronomer Cosmos committers.
+
+Committers have the same responsibilities as standard contributors and also perform the following actions:
+
+* Reviewing & merging pull-requests
+* Scanning and responding to GitHub issues, helping triaging them
+* Releasing new versions of Cosmos in `PyPI <https://pypi.org/project/astronomer-cosmos/>`_, following the `milestones <https://github.com/astronomer/astronomer-cosmos/milestones>`_.
+* Refactoring code
+* Improving processes and tooling
```
Comment (tatiana) [inline docs/contributors-roles.rst:41]:
```suggestion
```

File: docs/contributors-roles.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Pre-requisites to becoming a committer
+.......................................
+
+General prerequisites that we look for in all candidates:
+
+1. Consistent contribution over last few months
+2. Visibility on discussions on the Slack channel or GitHub issues/discussions
+3. Contributions to community health and project's sustainability for the long-term
+4. Understands the project's contributors guidelines :ref:`contributing`.
```
Comment (tatiana) [inline docs/contributors-roles.rst:55]:
```suggestion
4. Understands the project's contributors guidelines :ref:`contributing`.

Astronomer has the right to grant and revoke write access permissions to the project's official repository for any reason it sees fit.
```

File: docs/contributors-roles.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+* Fixing bugs
+* Refactoring code
+* Improving processes and tooling
+* Adding features
+* Improving the documentation
+* Making/answering questions in the #airflow-dbt Slack channel
+
+
+Committers/Maintainers
```
Comment (jlaneve) [inline docs/contributors-roles.rst:29]:
```suggestion
Committers
```

File: docs/contributors.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
+.. _contributors:
+Contributors
+============
+
+There are different ways people can contribute to Astronomer Cosmos.
+Learn more about the project contributors roles in :ref:`contributors-roles`.
+
+Committers/Maintainers
```
Comment (jlaneve) [inline docs/contributors.rst:8]:
```suggestion
Committers
```

File: docs/contributors.rst [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+Committers
+----------------------
+
+* Chris Hronek (`@chrishronek <https://github.com/chrishronek>`_)
+* Harel Shein (`@harels <https://github.com/harels>`_)
+* Julian LaNeve (`@jlaneve <https://github.com/jlaneve>`_)
+* Tatiana Al-Chueyr (`@tatiana <https://github.com/tatiana>`_)
+
+
+Emeritus Committers/Maintainers
```
Comment (jlaneve) [inline docs/contributors.rst:17]:
```suggestion
Emeritus Committers
```

**Review by jlaneve — APPROVED** [review]
added one more small comment (missed it during the first pass) but otherwise looks great, thank you for adding this!

**Pull request merged**